### ISSUE TITLE
construction/metadata with empty options

### ISSUE THREAD

**Issue by qiwu7** [Zondax/rosetta-filecoin-proxy#32]
Currently the options is required when querying construction/metadata
https://github.com/Zondax/rosetta-filecoin-proxy/blob/master/rosetta/services/construction.go#L60-L62

We should expand it to have options as optional. Since there are use cases where we just need to do fee estimation for the network, which does not need the address. Also as the fields suggested, `options` should be an optional field not `required` IMO. 

request
```
curl --location --request POST '{NODE_URL}/construction/metadata' \
--header 'Content-Type: application/json' \
--data-raw '{
    "network_identifier": {
        "blockchain": "Filecoin",
        "network": "testnet"
    }
}'
```
response
```
{
    "code": 0,
    "message": "ConstructionMetadataRequest.Options is nil",
    "retriable": false
}
```

**Comment by raynaudoe** [comment]
Ok!, I can add that... you can always send some dummy address to get a fee estimation, though. 

Also, it would be a good idea to clarify if `options` is an `optional` field, here:
```
// ConstructionMetadataRequest A ConstructionMetadataRequest is utilized to get information required
// to construct a transaction. The Options object used to specify which metadata to return is left
// purposely unstructured to allow flexibility for implementers.
type ConstructionMetadataRequest struct {
	NetworkIdentifier *NetworkIdentifier `json:"network_identifier"`
	// Some blockchains require different metadata for different types of transaction construction
	// (ex: delegation versus a transfer). Instead of requiring a blockchain node to return all
	// possible types of metadata for construction (which may require multiple node fetches), the
	// client can populate an options object to limit the metadata returned to only the subset
	// required.
	Options map[string]interface{} `json:"options"`
}
```

**Comment by qiwu7** [comment]
> Also, it would be a good idea to clarify if options is an optional field

Yeah I think it's a good idea to update the language there indicating that this is optional.

> Ok!, I can add that... you can always send some dummy address to get a fee estimation, though.

Getting the following error if I leave `Id` as empty or use invalid address....
{
    "code": 10,
    "message": "invalid account address",
    "retriable": false
}

**Comment by raynaudoe** [comment]
Fixed on #35
